src/sql/connection/connection.py

```diff
@@ -758,6 +758,9 @@ def _connection_execute(self, query, parameters=None):
 
     def _execute_with_parameters(self, query, parameters):
         """Execute the query with the given parameters"""
+        if parameters == "disable":
```

Comment:
In most functions this argument is `enable_named_params` but here it is passed as `parameters`. Do we want to add a new argument `enable_named_params` for consistency?

Reply:
I [removed enable_named_params](https://github.com/ploomber/jupysql/pull/980/files#diff-62e3d6289e3242b271e6df3a7f201ab003f8ce73a5daf1a824cc2b4d892513d1R759-R763) so now this operates on parameters itself.